sm_property_tree/include/sm/BoostPropertyTree.hpp

```diff
@@ -87,14 +87,23 @@ namespace sm {
     static bool isHumanReadableInputOutput() {
       return humanReadableInputOutput;
     }
+    static bool isIgnoreComments() {
+      return ignoreComments;
+    }
 
     static void setHumanReadableInputOutput(bool humanReadableInputOutput) {
       BoostPropertyTree::humanReadableInputOutput = humanReadableInputOutput;
     }
+    static void setIgnoreComments(bool ignoreComments) {
+      BoostPropertyTree::ignoreComments = ignoreComments;
+    }
+    
+    static int getXmlReadOptions();
```

Comment:
Ether that or move the function to BoostPropertyTreeImplementation by using the public static getters BoostPropertyTree::isHumanReadableInputOutput() and BoostPropertyTree::isIgnoreComments() That makes even more sense to me, because it is an implementation detail after all.